public/app/types/user.ts

```diff
@@ -131,3 +131,25 @@ export interface UserListAdminState {
   isLoading: boolean;
   sort?: string;
 }
+
+export interface UserAnonymousDeviceDTO {
+  login?: string;
+  client_ip: string;
+  device_id: string;
+  user_agent: string;
+  updated_at: string;
```

Comment:
Can we make all these fields camelCase to follow the [FE naming convention](https://github.com/grafana/grafana/blob/b022ddeee84225f99734b47613e5008c7e75ada4/contribute/style-guides/frontend.md#use-camelcase-for)?